WcaOnRails/app/models/reassign_wca_id.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+class ReassignWcaId
+  include ActiveModel::Model
+
+  attr_reader :account1, :account2
+  attr_reader :account1_user, :account2_user
+
+  def account1=(account1)
+    @account1 = account1
+    @account1_user = User.find_by_id(account1)
+  end
+
+  def account2=(account2)
+    @account2 = account2
+    @account2_user = User.find_by_id(account2)
+  end
+
+  validates :account1, presence: true
+  validates :account2, presence: true
+
+  validate :require_valid_accounts
+  def require_valid_accounts
+    if !account1
+      errors.add(:account1, "Not found")
+    end
+    if !account2
+      errors.add(:account2, "Not found")
+    end
+  end
+
+  validate :require_different_people
+  def require_different_people
+    if account1 == account2
```

Comment:
Nit: this should probably ignore the case when both `account1` and `account2` are `nil`.

Reply:
Sure! I added a `return unless errors.blank?` so it won't show the error if they are both nil (as the error for can't be blank will show instead).

Reply:
Yeah that's an idea, but it depends on the validation order, so I think it would be easier to reason about if you just check they are not `nil`, as you do for `must_look_like_the_same_person` :>

Reply:
Sounds good to me!